docs/es/week01/01-3.md

```diff
@@ -0,0 +1,241 @@
+<!--
+---
+lang-ref: ch.01-3
+title: Problem Motivation, Linear Algebra, and Visualization
+authors: Derek Yen, Tony Xu, Ben Stadnick, Prasanthi Gurumurthy
+---
+-->
+
+lang-ref: ch.01-3
+title: Motivación, Álgebra Lineal y Visualización \
+authors: Derek Yen, Tony Xu, Ben Stadnick, Prasanthi Gurumurthy
+lang: es\
+translation-date: 30 Mar 2020\
+translator: juanmartinezitm
+---
```

Comment:
You need to keep the same format here.
This file is read by a script, which expects a specific syntax, which starts with `---`.
Do not add `\` as well.
Also, where's the original `date`?